blocks/library/cover-image/index.js

```diff
@@ -152,22 +177,69 @@ registerBlockType( 'core/cover-image', {
 				className={ classes }
 			>
 				{ title || !! focus ? (
-					<Editable
-						tagName="h2"
-						placeholder={ __( 'Write title…' ) }
-						value={ title }
-						focus={ focus }
-						onFocus={ setFocus }
-						onChange={ ( value ) => setAttributes( { title: value } ) }
-						inlineToolbar
+					<div className="title-container">
```

Comment:
Trying to understand the need for `.title-container` and absolute positioning: It seems like it's a workaround to the real issue, which is that the flexbox behavior of the cover image block doesn't stack the two adjacent children well otherwise, which can probably be addressed with flex styles such as `flex-direction: column;`